torch/_C/__init__.pyi.in

```diff
@@ -413,10 +413,13 @@ class ErrorReport:
     def call_stack() -> str: ...
 
 class CompilationUnit:
-    def __init__(self) -> None: ...
+    def __init__(self, lang: str=..., _frames_up: _int=...) -> None: ...
     def find_function(self, name: str) -> ScriptFunction: ...
```

Comment:
Is this still accurate? Should it be `Optional[ScriptFunction` since the new implementation can return `None` now?
```suggestion
    def find_function(self, name: str) -> Optional[ScriptFunction]: ...
```

Reply:
:) Right.